app/models/user.rb

```diff
@@ -29,7 +30,11 @@ def reset_password(new_password, new_password_confirmation)
     if new_password.present?
       self.password = new_password
       self.password_confirmation = new_password_confirmation
-      save!
+      if save
+        true
+      else
+        false
```

Comment:
Just wondering when this will happen, that save fails, and will the user hear about it ?

Reply:
Yea, not a super fan of this, but more elegant ways tiggered rubocop, so doing this for now (preferred solution would be not to have admin updating passwords, so should go away once we can correct that -- for now, ugly code serves as a warning)